CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
 
+# 0.53.0
+
+## Upgrade Instructions
+
+
+Whiskey no longer installs PackageManagement or PowerShellGet modules (again). On all computers running Whiskey builds
+install a version of:
+
+* PackageManagement from 1.3.2 to 1.4.8.1

```

Comment:
The maximum version is actually `1.4.7` in Whiskey.psm1

Reply:
Those variables aren't being used anymore. I'll remove them.